lib/utils/logger.ts

```diff
@@ -58,6 +58,7 @@ export class Logger {
 	// These can be overriden by specific methods.
 	private _logLevel = _.parseInt(process.env.PROCBOT_LOG_LEVEL) || LogLevel.INFO;
 	private _alertLevel = _.parseInt(process.env.PROCBOT_ALERT_LEVEL) || AlertLevel.ERROR;
+	private _secrets: string[] = [];
```

Comment:
This doesn't need to be `_secrets` as there's no setter/getter with the same name.

Reply:
Note to self: ignored as per next comment.